pkg/helm/cache.go

```diff
@@ -0,0 +1,81 @@
+// Copyright © 2018 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helm
+
+import (
+	"time"
+
+	"github.com/patrickmn/go-cache"
+	"github.com/sirupsen/logrus"
+	rls "k8s.io/helm/pkg/proto/hapi/services"
+)
+
+var globalCache *cache.Cache
+
+func init() {
+	globalCache = cache.New(1*time.Minute, 5*time.Minute)
+}
+
+type releaseCache interface {
```

Comment:
If this is going to be a general releaseCache interface for future cache implementations, maybe error handling should be added (I'm not sure though), or just in case of an error we should return cache-miss (?).